example/src/main/java/org/eclipse/hono/example/ExampleReceiver.java

```diff
@@ -79,7 +79,7 @@ private void onDisconnect(final ProtonConnection con) {
 
         // give Vert.x some time to clean up NetClient
         vertx.setTimer(DEFAULT_CONNECT_TIMEOUT_MILLIS, reconnect -> {
-            LOG.info("attempting to re-connect to Hono server");
+            LOG.info("attempting to re-connect to Hono Messaging");
```

Comment:
Yes, you are right. But then Hono server would be not correct either, or WDYT?
Should it be "Hono Messaging Network"? Or "Hono AMQP 1.0 Messaging Network"?